blueocean-pipeline-scm-api/src/main/java/io/jenkins/blueocean/scm/api/AbstractPipelineCreateRequest.java

```diff
@@ -72,4 +72,8 @@ protected ModifiableTopLevelItemGroup getParent() {
 
         return parent;
     }
+
+    protected BlueOrganization findOrganization() {
+        return OrganizationFactory.getInstance().get(getOrganization());
```

Comment:
@alvarolobato This won't work with multi-org case. What we need is to update [BluePipelineCreateRequest.create()](https://github.com/jenkinsci/blueocean-plugin/blob/8d8d5234579167ebbc072afda09d554d002ec8b0/blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BluePipelineCreateRequest.java#L41) and pass BlueOrganization parameter. Then pass organization when calling create: https://github.com/jenkinsci/blueocean-plugin/blob/4cefa8d4a94bb513d252e436ae8228f809368209/blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BluePipelineContainer.java#L54.